weave

```diff
@@ -261,10 +261,10 @@ docker_bridge_ip() {
 
 # the following borrows from https://github.com/jpetazzo/pipework
 
-# Set $NETNS to the network namespace of container $1, $LOCAL_IFNAME
-# and $GUEST_IFNAME to suitable names for two ends of a veth pair,
-# specific to the container, and execute args $2 $3 ... as a command.
-# If an error is caused by container dying, swallow output from error.
+# Set $LOCAL_IFNAME and $GUEST_IFNAME to suitable names for two ends
+# of a veth pair, specific to the container, and execute args $2 $3 ...
+# as a command. If an error is caused by container dying, swallow output
+# from error.
```

Comment:
Should note here that this  sets `CONTAINER_PID` (to the pid of the container passed in $1).